.github/workflows/cd.yaml

```diff
@@ -0,0 +1,24 @@
+name: CD
+
+on:
+  push:
```

Comment:
Should this be a `create` trigger instead?   A `pypi` release should be immutable to avoid unexpected breaking changes for our users, so we don't want to republish a tag that we've already published to `pypi`.  In case a tag accidentally gets overwritten or something.

Reply:
Nice catch